frontend/models/response.tsx

```diff
@@ -0,0 +1,25 @@
+import { AppRoutes } from "./"
+
+interface EnrollmentErrorText {
+  statusMessage: string
+  returnText: string
+  returnPath: string
+}
+
+export enum EnrollmentTypes {
+  VIEWER = "viewer",
+  PASSPORT = "passport"
+}
+
+export const enrollmentMessage: { [key in EnrollmentTypes]: EnrollmentErrorText } = {
```

Comment:
For most literal values, you don't need to specify the type as the compiler can infer it. but this does prevent people from adding unrelated keys.